src/imitation/scripts/train_adversarial.py

```diff
@@ -36,102 +38,151 @@ def save(trainer, save_path):
-        save weights at all.
+        _seed: Random seed.
+        algorithm: A case-insensitive string determining which adversarial imitation
+            learning algorithm is executed. Either "airl" or "gail".
+        env_name: The environment to train in.
+        num_vec: Number of `gym.Env` to vectorize.
+        parallel: Whether to use "true" parallelism. If True, then use `SubProcVecEnv`.
+            Otherwise, use `DummyVecEnv` which steps through environments serially.
+        max_episode_steps: If not None, then a TimeLimit wrapper is applied to each
+            environment to artificially limit the maximum number of timesteps in an
+            episode.
+        rollout_path: Path to pickle containing list of Trajectories. Used as
+            expert demonstrations.
+        n_expert_demos: The number of expert trajectories to actually use
+            after loading them from `rollout_path`.
+            If None, then use all available trajectories.
+            If `n_expert_demos` is an `int`, then use exactly `n_expert_demos`
+            trajectories, erroring if there aren't enough trajectories. If there are
+            surplus trajectories, then use the first `n_expert_demos` trajectories and
+            drop the rest.
+        log_dir: Directory to save models and other logging to.
+        total_timesteps: The number of transitions to sample from the environment
+            during training.
+        n_episodes_eval: The number of episodes to average over when calculating
+            the average episode reward of the imitation policy for return.
+        init_tensorboard: If True, then write tensorboard logs to `{log_dir}/sb_tb`.
+        checkpoint_interval: Save the discriminator and generator models every
+            `checkpoint_interval` epochs and after training is complete. If 0,
+            then only save weights after training is complete. If <0, then don't
+            save weights at all.
+        gail_discrim_net_scale: If True, then normalize observation inputs coming into
+            the `DiscrimNetGAIL` using the bounds of the environment's observation
+            space. Argument is ignored when training AIRL.
+        airl_entropy_weight: The entropy weight for AIRL training reward. Argument is
+            ignored when training GAIL.
+        airl_reward_net_cls: The `RewardNet` class to instantiate when initializing the
+            `DiscrimNetAIRL`. Argument is ignored when training GAIL.
+        airl_reward_net_kwargs: Keyword arguments to pass into the
+            `RewardNet` constructor. Passing in `action_space` and `observation_space`
+            results in error because they are automatically inferred from `venv`.
+        init_rl_kwargs: Keyword arguments for `init_rl`, the RL algorithm initialization
+            utility function.
+        algorithm_kwargs: Keyword arguments for the `GAIL` or `AIRL` constructor that
```

Comment:
It would be more symmetric if you had `gail_algorithm_kwargs` and `airl_algorithm_kwargs` in addition to the global `algorithm_kwargs`, and I think would simplify some of the subsequent logic.